### DISCUSSION TITLE
Is there a property or an event to tell if Scene Viewer was launched ?

### DISCUSSION THREAD

**Discussion by ryancrunchi** [google/model-viewer#2334] [Q&A]
I know I can give mode priority with `ar-modes="scene-viewer webxr"` or `ar-modes="webxr scene-viewer"`.
But even in this latter case, the webxr can fails sometimes and launch Scene Viewer as a fallback. I need to know if scene viewer has been launched.
The `ar-status` event tell if it failed, but don't tell which mode.
Is it possible ?

Maybe at a higher overview, having an event or a property for which mode has been **effectively** activated is relevant ?

**Comment by elalish** [comment]
It's a bit tricky, as we have no idea what the status is once we intent out to scene-viewer, so we can't tell if that failed. What are you trying to accomplish by differentiating? You could restrict to `ar-modes="webxr"` and then handle the fallback manually.
